website/content/api-docs/system/internal-counters.mdx

```diff
@@ -263,6 +263,10 @@ not have namespace attribution for the `new_clients` stanza. Furthermore, the
 `new_clients` counts returned for the current month will be an approximation.
 That is to say, the response will appear as follows.
 
+- As of 1.15.13, 1.16.7, 1.17.3, 1.18 and later, the billing start date (license start date if not configured) is the auto-rolled to the most recent year.
```

Comment:
This paragraph comes in between `That is to say, the response will appear as follows` and the example that's being referenced. Can we move this paragraph somewhere else in the page?

Reply:
I will move this as a note in "Read the client count configuration"